en/docs/install-and-setup/setup/reference/default-product-ports.md

```diff
@@ -1,347 +1,105 @@
 
-[Common ports](#common-ports) | [Management console ports](#management-console-ports) | [LDAP server ports](#ldap-server-ports) | [KDC ports](#kdc-ports) | [JMX monitoring ports](#jmx-monitoring-ports) | [Clustering ports](#clustering-ports) | [Random ports](#random-ports)
-
-#### Management console ports
+### Management console ports
 
 WSO2 products that provide a management console use the following servlet transport ports:
 
 -   9443 - HTTPS servlet transport (the default URL of the management console is https://localhost:9443/carbon )
 -   9763 - HTTP servlet transport
 
-WSO2 API Manager (WSO2 APIM) uses the following ports to access the management console:
-
--   9443 - HTTPS servlet transport for the **APIM** runtime (the default URL of the management console is https://localhost:9443/carbon )
--   9444 - Used for the **APIM-Analytics** management console
-
-#### LDAP server ports
+### LDAP server ports
 
 Provided by default in the WSO2 Carbon platform.
 
 -   10389 - Used in WSO2 products that provide an embedded LDAP server
 
-#### KDC ports
+### KDC ports
 
 -   8000 - Used to expose the Kerberos key distribution center server
 
-#### JMX monitoring ports
+### JMX monitoring ports
 
 WSO2 Carbon platform uses TCP ports to monitor a running Carbon instance using a JMX client such as JConsole. By default, JMX is enabled in all products. You can disable it by adding following configuration to `<PRODUCT_HOME>/repository/conf/deployment.toml` file.
 
-``` java
-    [monitoring.jmx]
-    rmi_server_start = false
+``` toml
+[monitoring.jmx]
+rmi_server_start = false
 ```
 
 -   11111 - RMIRegistry port. Used to monitor Carbon remotely
 -   9999 - RMIServer port. Used along with the RMIRegistry port when Carbon is monitored from a JMX client that is behind a firewall
 
-#### Clustering ports
+### Clustering ports
 
 To cluster any running Carbon instance, either one of the following ports must be opened.
 
 -   45564 - Opened if the membership scheme is multicast
 -   4000 - Opened if the membership scheme is wka
 
-#### Random ports
+### Random ports
 
 Certain ports are randomly opened during server startup. This is due to specific properties and configurations that become effective when the product is started. Note that the IDs of these random ports will change every time the server is started.
 
 -   A random TCP port will open at server startup because of the `-Dcom.sun.management.jmxremote` property is set in the server startup script. This property is used for the JMX monitoring facility in JVM.
--   A random UDP port is opened at server startup due to the log4j appender ( `SyslogAppender` ), which is configured in the `<PRODUCT_HOME>/repository/conf/log4j.properties` file.
-
-### Product-specific ports
 
-Some products open additional ports.
+-   A random UDP port is opened at server startup due to the log4j appender ( `SyslogAppender` ), which is configured in the `<PRODUCT_HOME>/repository/conf/log4j2.properties` file.
```

Comment:
When adding brackets do not leave a space after the bracket opens and before the bracket closes.